modules/cnv-virtctl-commands.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-using-the-cli-tools.adoc
+
+[id="cnv-virtctl-commands_{context}"]
+= Virtctl client commands
+
+The `virtctl` client is a command-line utility for managing {ProductName}
+resources. The following table contains the `virtctl` commands used throughout
+the {ProductName} documentation.
+
+.Virtctl client commands
+
+[width="100%",cols="42%,58%",options="header",]
+|===
+|Command |Description
+
+|`virtctl start <vm>` 
+|Start a virtual machine, creating a virtual machine instance.
```

Comment:
Please don't mention "instance" in the context of start/stop/restart. I think that confuses things quite a bit. For reference, here's the help output from (the devel version of) virtctl:

```
virtctl controls virtual machine related operations on your kubernetes cluster.

Available Commands:
  console      Connect to a console of a virtual machine instance.
  expose       Expose a virtual machine instance, virtual machine, or virtual machine instance replica set as a new service.
  help         Help about any command
  image-upload Upload a VM image to a PersistentVolumeClaim.
  restart      Restart a virtual machine.
  start        Start a virtual machine.
  stop         Stop a virtual machine.
  version      Print the client and server version information.
  vnc          Open a vnc connection to a virtual machine instance.

Use "virtctl <command> --help" for more information about a given command.
Use "virtctl options" for a list of global command-line options (applies to all commands).
```

Reply:
Glad to hear it. I've long found it awkward to try to differentiate VM from VMI in this context when writing the docs, especially for users from traditional virtual backgrounds. 
This will impact other docs, so is it safe to say that VMI should not be referred to in any context of the control (start/stop/restart)?

Reply:
I hesitate to generalize too much. When speaking of virtctl in particular the start/stop/restart operations are at the VM level--but in other contexts it might make sense to refer to start/stop and the VMI level... so it depends.